base/linalg/dense.jl

```diff
@@ -343,31 +345,16 @@ function ^{T}(A::AbstractMatrix{T}, p::Real)
         retmat = retmat * full(powm(UpperTriangular(float.(A)), real(p - floor(p))))
         d = diag(A)
     else
-        S,Q,d = schur(complex(A))
+        S,Q,d = schur(complex(full(A)))
```

Comment:
It should, I guess.

Because I need the complex schur form which is different for e.g. `A = [0 1; -1 0]`